android/jni/app-android.cpp

```diff
@@ -814,6 +814,10 @@ extern "C" jboolean Java_org_ppsspp_ppsspp_NativeApp_joystickAxis(
 	axis.axisId = axisId;
 	axis.deviceId = deviceId;
 	axis.value = value;
+
+	float sensitivity = g_Config.fXInputAnalogSensitivity;
+	axis.value *= sensitivity;
```

Comment:
I started typing thinking I was going to multiply two axis with the value I guess, but there was only one. Doesn't really matter.